kms/api/Kms.Samples/VerifyAsymmetricEc.cs

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START kms_verify_asymmetric_signature_ec]
+using Google.Cloud.Kms.V1;
+
+public class VerifyAsymmetricSignatureEcSample
+{
+
+    public bool VerifyAsymmetricSignatureEc(
+      string projectId = "my-project", string locationId = "us-east1", string keyRingId = "my-key-ring", string keyId = "my-key", string keyVersionId = "123",
+      string message = "my message",
+      byte[] signature = null)
+    {
+        // Cloud KMS returns signatures in a DER-encoded format. .NET requires
```

Comment:
I'll pre-empt a question here. I'd really really really prefer to include this. We get a lot of feedback and bug reports on "please include a sample for X". Often times, we haven't _forgotten_ to include a sample, but it's actually not possible with the current language primitives (or would require hundreds of lines of code, 800 in this case). I'd rather users see this warning than think we forgot.

Reply:
This is totally fine by me. I wouldn't usually mess with the content of a sample unless I find it too weird.
Only thing I would do is remove the method signature, the class etc. Basically just have a text file with the tags and the comment. If it need to be a .cs file then leave the comments as //, else just a .txt file with this text in between tags. Less code to maitan or that people can add their own things inadvertently.

Reply:
I have to keep the class (it gets sad otherwise), but I reduced it just to a comment.